### ISSUE TITLE
Units to be refined not showing up by default unless in a larger unit

### ISSUE THREAD

**Issue by nschneid** [omriabnd/UCCA-App#17]
When I load task 620, the supersense layer refining the P category, I see just 3 units listed below the text. These are apparently the weak MWEs and their child units (strong expressions). I would have expected all and only units requiring refinement to be listed.

Is this a bug or a problem with the layer configuration?

**Comment by omriabnd** [comment]
This is just the display. You can open any unit you like by double-clicking
on it in the passage level.
I'm not sure why exactly these three units are the ones listed. Would it be
better if they were all closed upon loading, and that you would open only
the ones you're interested in?

**Comment by nschneid** [comment]
I don't think it makes sense to force the user to double-click on every unit to be refined, given that it is known which units those are (the task can't be submitted until the refinement category is chosen).
